files/msgdata/msg/pokemon_summary_screen.gmm

```diff
@@ -97,343 +97,343 @@
 	</row>
-	<row id="msg_0302_00093" index="93">
+	<row id="PokemonSummary_Text_ThoroughlyCunning" index="93">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Thoroughly cunning.</language>
 	</row>
-	<row id="msg_0302_00094" index="94">
+	<row id="PokemonSummary_Text_OftenLostInThought" index="94">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Often lost in thought.</language>
 	</row>
-	<row id="msg_0302_00095" index="95">
+	<row id="PokemonSummary_Text_VeryFinicky" index="95">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Very finicky.</language>
 	</row>
-	<row id="msg_0302_00096" index="96">
+	<row id="PokemonSummary_Text_StrongWilled" index="96">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Strong willed.</language>
 	</row>
-	<row id="msg_0302_00097" index="97">
+	<row id="PokemonSummary_Text_SomewhatVain" index="97">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Somewhat vain.</language>
 	</row>
-	<row id="msg_0302_00098" index="98">
+	<row id="PokemonSummary_Text_StronglyDefiant" index="98">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Strongly defiant.</language>
 	</row>
-	<row id="msg_0302_00099" index="99">
+	<row id="PokemonSummary_Text_HatesToLose" index="99">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Hates to lose.</language>
 	</row>
-	<row id="msg_0302_00100" index="100">
+	<row id="PokemonSummary_Text_SomewhatStubborn" index="100">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">Somewhat stubborn.</language>
 	</row>
-	<row id="msg_0302_00101" index="101">
+	<row id="PokemonSummary_Text_MetCondition_Egg" index="101">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">{STRVAR_34 16, 6, 0} {STRVAR_1 51, 7, 0}, 20{STRVAR_1 51, 5, 0}\nA mysterious Pokémon\nEgg received from\n{COLOR 2}{STRVAR_1 4, 8, 0}{COLOR 0}.\n</language>
 	</row>
-	<row id="msg_0302_00102" index="102">
+	<row id="PokemonSummary_Text_MetCondition_EggTraded" index="102">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">{STRVAR_34 16, 6, 0} {STRVAR_1 51, 7, 0}, 20{STRVAR_1 51, 5, 0}\nA mysterious Pokémon\nEgg obtained in\n{COLOR 2}{STRVAR_1 4, 8, 0}{COLOR 0}.\n</language>
 	</row>
-	<row id="msg_0302_00103" index="103">
+	<row id="PokemonSummary_Text_MetCondition_FatefulEgg" index="103">
 		<attribute name="window_context_name">used</attribute>
 		<language name="English">{STRVAR_34 16, 6, 0} {STRVAR_1 51, 7, 0}, 20{STRVAR_1 51, 5, 0}\nA very mysterious\nPokémon Egg obtained\nat {COLOR 1}{STRVAR_1 4, 8, 0}{COLOR 0}.\n</language>
 	</row>
-	<row id="msg_0302_00104" index="104">
+	<row id="PokemonSummary_Text_MetCondition_FatefulEggArrived" index="104">
```

Comment:
I'd like a second opinion on message names, either I'd want Text_ to come first or have the prefix be Msg_ instead since this _is_ in the msg folder